espnet/bin/asr_recog.py

```diff
@@ -174,6 +174,13 @@ def get_parser():
         "--ngram-model", type=str, default=None, help="ngram model file to read"
     )
     parser.add_argument("--ngram-weight", type=float, default=0.1, help="ngram weight")
+    parser.add_argument(
+        "--ngram-scorer",
+        type=str,
+        default="full",
+        choices=("full", "part"),
+        help="ngram scorer choices",
```

Comment:
Could you explain a bit more about the full and part scorer options?

Reply:
If the ngram is set as a part scorer, similar to CTC scorer, the ngram scorer only score to topID.
but ngram is set as a full scorer, the ngram score all ID.

Reply:
Oh sorry, I meant to say that it’s better to add such descriptions. Probably in the help message?

Reply:
good idea, I will add it.